doc/changes/devel/12850.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug introduced in :gh:`12850` where the pupil channel SI unit was incorrectly set to Moles instead of Meters. (by `Scott Huberty`_)
```

Comment:
```suggestion
Fix bug introduced in :gh:`12846` where the pupil channel SI unit was incorrectly set to Moles instead of Meters. (by `Scott Huberty`_)
```